pool/src/storage.rs

```diff
@@ -9,6 +9,7 @@ pub enum DataKey {
     ReserveAssetKey(Address),
     Reserves,
     UserConfig(Address),
+    OraclePriceFeed,
```

Comment:
I think we should save oracle feed for every underlying asset. Could it be better to save oracle address under `ReserveAssetKey(Asset)`

Reply:
Also we can store Option<Address> in ReserveData to ommit oracle initialization in init_reserve

Reply:
Updated to support asset to oracle relation.